pkg/dedup/iter.go

```diff
@@ -23,6 +23,12 @@ import (
 // that timestamps are in milliseconds and sampling frequencies typically multiple seconds long.
 const initialPenalty = 5000
 
+const (
+	AlgorithmPenalty = "penalty"
```

Comment:
These three dedup will change the alert behaviors, how can we validate in the future to avoid regression when updating this?

Reply:
we should use `quorum` all the time, people shouldn't make change and need to go through shard flags and rigorous testing